packages/config/config/devices/0x0460/qnsw-001P16.json

```diff
@@ -73,102 +99,220 @@
 		{
 			"#": "23",
 			"label": "O1 No/Nc",
-			"description": "0",
+			"description": "The set value determines the relay contact type for output O (O1). The relay contact type can be normally open (NO) or normally closed (NC).",
```

Comment:
With the updated label and the predefined options below, this description isn't necessary.